react/components/SearchResultContainer.js

```diff
@@ -107,7 +108,15 @@ export default class SearchResultContainer extends Component {
     )
     // Need to redo this to calculate the rest and map in the pageProps
     restParam && (pageProps.query.rest = restParam)
-    mapParam && (pageProps.query.map = mapParam)
+    if (mapParam) {
```

Comment:
please, remove these lines, don't hardcode this, it should be handled only by the `getPagesArgs` function